prism/src/main/java/org/apache/falcon/resource/proxy/ExtensionManagerProxy.java

```diff
@@ -317,8 +330,8 @@ public APIResult submit(
 
         try {
             entityMap = getEntityList(extensionName, jobName, feedForms, processForms, config);
-            submitEntities(extensionName, doAsUser, jobName, entityMap, config);
-        } catch (FalconException | IOException e) {
+            submitEntities(extensionName, jobName, entityMap, config, request);
```

Comment:
Why has doAsUser been knocked off? I know we don't use it now. But, since the plan to enable it later, we shouldn't be removing it.